app/jobs/in_person/send_proofing_notification_job.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+module InPerson
+  class SendProofingNotificationJob < ApplicationJob
+    # @param [String] enrollment_id primary key of the enrollment
+    def perform(enrollment_id)
+      return true if IdentityConfig.store.in_person_proofing_enabled.blank? ||
```

Comment:
I think having a return value in an async job is not useful, I'd recommend just making all of these `return`?

Reply:
@zachmargolis  fixed.